sdks/Cordova.md

```diff
@@ -0,0 +1,138 @@
+# Apache Cordova AeroGear Services
+
+## Motivation
+
+Provide set of the SDK's that can be used for Cordova based development.
+Provide best performance and user experience on the platform.
+
+## Proposed solution 1
+
+Pure javascript solution for the mobile configuration and orchestration 
+that will allow cooperability between JavaScript and Native.
+Mixing both existing Native plugins and pure JavaScript based solutions was identified as the most reasonable aproach.
```

Comment:
I think this line is still a bit confusing for me. Does it mean we always have a native & js implementation, or we should look at the possibility of having a pure JS implementation first?

I think it needs wording a little bit better to avoid any confusion.

Reply:
Thank you. I will correct that to make sure that we mean JS first and plugin when it is not possible/wrapper is needed.